src/style.scss

```diff
@@ -0,0 +1,135 @@
+$main__color: #060b35;
+$secondary__color: #616070;
+$button__color: #00acdc;
+$white__color: #fff;
```

Comment:
You shouldn't name variables by content, but you should name variables by meaning, because in the process of development, the color may change and you will have to change it throughout the code.